Octokit/Http/ReadOnlyPagedCollection.cs

```diff
@@ -28,7 +28,14 @@ public ReadOnlyPagedCollection(IApiResponse<List<T>> response, Func<Uri, Task<IA
             var nextPageUrl = _info.GetNextPageUrl();
             if (nextPageUrl == null) return null;
 
-            var response = await _nextPageFunc(nextPageUrl).ConfigureAwait(false);
+            var maybeTask = _nextPageFunc(nextPageUrl);
+
+            if (maybeTask == null)
```

Comment:
Dont feel too bad :grinning: It seems like an acceptable compromise in order to implement the "bail out" without having to totally change the entire way Octokit.net has looped through all the paged responses until now...